frontend/app/components/Spots/SpotPlayer/components/AccessModal.tsx

```diff
@@ -132,68 +132,74 @@ function AccessModal() {
       {!isPublic ? (
         <>
           <div>
-            <div className={'text-disabled-text'}>
-              All team members in your project will able to view this Spot
+            <div className={'text-black/50'}>
+              Link for internal team members
             </div>
-            <div className={'px-2 py-1 border rounded bg-[#FAFAFA] whitespace-nowrap overflow-ellipsis overflow-hidden'}>
+            <div className={'px-2 py-1 rounded-lg bg-indigo-50 whitespace-nowrap overflow-ellipsis overflow-hidden'}>
               {spotLink}
             </div>
           </div>
           <div className={'w-fit'}>
             <Button
               size={'small'}
               onClick={onCopy}
-              type={'text'}
-              icon={<LinkOutlined />}
+              type={'default'}
+              icon={<CopyOutlined />}
             >
               {isCopied ? 'Copied!' : 'Copy Link'}
             </Button>
           </div>
         </>
       ) : !generated ? (
-        <div className={'w-fit'}>
+        <div className={'w-fit p-1'}>
           <Button
-            loading={spotStore.isLoading}
+            loading={loadingKey}
```

Comment:
why remove one we have to introduce new one?

Reply:
using `spotStore.isLoading` is causing issue with `formatExpirationTime` so I've introduced new one which is local to this component.